README.md

```diff
@@ -25,136 +25,37 @@
 ```
-
-### Configure and Build
-
-4C enforces an out-of-source build, i.e., the build directory may not be the top-level source directory.
-Create a build directory `<path/to/build/dir>` of your choosing.
-
-Navigate into the build directory and run
-
-```bash
-cmake --preset=<name-of-preset> <path/to/source/dir>
+docker run --interactive --tty ghcr.io/4c-multiphysics/4c:main
+/home/user/4C/build/4C ../tests/input_files/<some-input-file>.4C.yaml output_name
 ```
 
-A preset name needs to be passed to cmake via the command line argument `--preset`.
-Use `cmake <path/to/source/dir> --list-presets` to get a list of all available presets.
-Define your own presets in a `CMakeUserPresets.json` file.
-
-More information about CMake presets can be found in the [CMake documentation](https://cmake.org/cmake/help/latest/manual/cmake-presets.7.html).
-
-#### Build
-
-To build the code on `<numProcs>` processes, run
-
-```bash
-ninja -j <numProcs> full
-```
-
-#### Run the Tests
-
-To verify that the build was successful, run the minimal set of tests via
-
-```bash
-ctest -L minimal
-```
-
-or all tests via
-
-```bash
-ctest
-```
-
-You can use the option `-j <num_threads>` to specify the number of threads to be used for parallel execution.
-
-### Prepare and Run Simulations
-
-After successfully building 4C, the executable `4C` is located in your build directory `<path/to/build/dir>/`.
-It needs to be invoked together with an input (`.dat`) file via
-
-```bash
-<path/to/build/dir>/4C <jobName>.dat <outputName>
-```
-
-where `<jobName>` is the name of the simulation file and `<outputName>` denotes the name of the corresponding output
-file(s).
-A collection of working `.dat` files is located under `tests/input_files/` in the source code repository.
-
-Input files (`*.dat`) can be generated through various mechanisms.
-Please consult our [user guide](https://4c-multiphysics.github.io/4C/readthedocs/index.html) for further information and detailed instructions.
-
-4C can write its simulation output in different formats.
-The [user guide](https://4c-multiphysics.github.io/4C/readthedocs/index.html) outlines the different output format and their necessary steps to actually view the results.
-
-## Where to Ask Questions
-
-If you need help with 4C, feel free to ask questions
-by [creating a GitHub issue](https://github.com/4C-multiphysics/4C/issues). Use an issue template to pre-populate the *Description* field, giving you instructions on submitting the issue.
-
-For more general questions, you can reach us on the [4C Slack workspace](https://join.slack.com/t/4c-multiphysics/shared_invite/zt-1oi61jgdd-5tZuHku3Tb_BH5UBgojbpQ).
+See our [documentation](https://4c-multiphysics.github.io/4C/readthedocs/installation.html) for more information on how to build 4C from source on your machine.
+Condult our [Tutorials](https://4c-multiphysics.github.io/4C/readthedocs/tutorials.html) to get an overview of the general workflow in 4C.
```

Comment:
```suggestion
Consult our [Tutorials](https://4c-multiphysics.github.io/4C/readthedocs/tutorials.html) to get an overview of the general workflow in 4C.
```

Reply:
I thought, I just accept your suggestion to avoid any mistakes from copy-paste. I should have read the suggestion better :sweat_smile: